### DISCUSSION TITLE
Add delay to trigger recognition to allow currently conflicting triggers

### DISCUSSION THREAD

**Discussion by harristom** [espanso/espanso#1510] [Feature requests and Ideas]
Right now the following doesn't work because the triggers conflict so only the first will run:

```
  - trigger: ":foo"
    replace: "foobar"

  - trigger: ":foobar"
    replace: "baz"
```

It would be great if you could add a delay to processing the trigger (possibly with trigger_delay option) to allow to complete typing the second trigger before processing it.

**Comment by smeech** [comment]
Duplicate of #1168?
